bindata/assets/alerts/client-go-requests.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: client-go
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+    - name: control-plane-client-go
+      rules:
+        - record: client_go_request_duration:method_get:histogram_quantile99
```

Comment:
naming suggestion:
```
cluster:rest_client_request_duration_seconds:histogram_quantile
```
essentially the format is: `<scope>:<metric>:<aggregation>`

Reply:
done